test/integration/navbar_links_test.rb

```diff
@@ -48,4 +19,26 @@ class NavbarLinksTest < ActionDispatch::IntegrationTest
       refute has_link? 'CM Resources'
     end
   end
+  
+  describe 'user dropdown' do
+    before { click_link "#{@user.name}" }
```

Comment:
Same deal here, you can probably just use `@user.name` instead of the interpolation.